interface/components/qqplot/index.coffee

```diff
@@ -0,0 +1,284 @@
+
+require "./index.styl"
+c3 = require "c3"
+Model = require "../Model"
+
+mean = (values) ->
+  sum = 0
+  i = 0
+  while i < values.length
```

Comment:
What is values.length is zero? This will cause an error. You need to add a check if values.length == 0